CellEditorIDFixer/CellEditorIDFixer.cs

```diff
@@ -39,7 +39,7 @@ public static void RunPatch(SynthesisState<ISkyrimMod, ISkyrimModGetter> state)
                 var cell = cellContext.Record;
                 if ((cell.EditorID?.Contains("_") ?? false))
                 {
-                    Console.WriteLine($"Cell EDID {cell.EditorID} in {cell?.FormKey.ModKey.FileName}");
```

Comment:
This one is funny, as cell is not nullable at this point.. but adding the unnecessary `?` makes the compiler complain on later lines that cell might be null.  Almost as if you checking unnecessarily makes it doubt itself "maybe it actually could be null??"